habitat_baselines/rl/requirements.txt

```diff
@@ -0,0 +1 @@
+torch>=1.0.0
```

Comment:
It will be better if you fix the torch version. The inconsistencies and breaking changes in PyTorch version has broken pythia more than once.